docs/Glossary.md

```diff
@@ -6,13 +6,13 @@
   environment.
 * **Agent** - Unity Component which produces observations and takes actions in
   the environment. Agents actions are determined by decisions produced by a
-  linked Brain.
-* **Brain** - Unity Asset which makes decisions for the agents linked to it.
-* **Decision** - The specification produced by a Brain for an action to be
+  Policy.
+* **Policy** - The decision making mechanism, typically a neural network model.
```

Comment:
Behavior is used a lot as @chriselion pointed out already.  Is there a reason to have both policy and behavior?

Reply:
Behavior is how I named the Agent's Parameters. Policy is the object that makes the decision and will eventually exist without Agents. So I wanted to separate the two to make it easier later to change the documentation.